SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -158,6 +158,9 @@ class CFVMFlowSolverBase : public CSolver {
   su2double** CPressureTarget = nullptr;   /*!< \brief Target Pressure coefficient for each boundary and vertex. */
   su2double** YPlus = nullptr;             /*!< \brief Yplus for each boundary and vertex. */
 
+  su2double** UTau = nullptr;              /*!< \brief UTau for each boundary and vertex. */
+  su2double** EddyViscWall = nullptr;      /*!< \brief Wall Eddy Viscosity for each boundary and vertex. */
+
```

Comment:
I changed all these things in my blitz PR, you should be able to replace su2double** by vector<vector<su2double> >, delete the destructor part and everything should work.

Reply:
Thanks for the remark, I don't like to work with ********variables either: we have c++ for that after all. Personally, I'd like to see a complete move from new..delete to vector-only (but I conform to the local programming standard as far as that exists). So can we say that it is best practice to replace all su2double* and su2double** to vector and vector<vector> ?

Reply:
vector vector if the the number of columns varies, otherwise su2activematrix which is all contiguous memory (vector vector is as bad for performance as **).

Reply:
Oh and please never conform ;)